manifest.json

```diff
@@ -2,7 +2,7 @@
   "vendor": "vtex",
   "name": "styleguide",
   "title": "VTEX Styleguide",
-  "version": "9.111.4",
+  "version": "9.111.5-beta",
```

Comment:
Do you have any specific reason for this to be a beta tag?

Reply:
test size in bundlephobia

https://bundlephobia.com/result?p=@vtex/styleguide@9.111.5-beta

Reply:
but `releasy` has a problem, he publish beta as latest!

Reply:
![Screen Shot 2020-02-17 at 19 28 52](https://user-images.githubusercontent.com/10627086/74689911-bffd3c80-51bb-11ea-8d33-4a856b2e20f7.png)